File: class_member/3.py
class Cs:
    #class 안, method 밖에 변수를 선언하면 class 소속 변수
    count = 0
    def __init__(self):
        Cs.count = Cs.count + 1
    @classmethod
    def getCount(cls):
        return Cs.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
i4 = Cs()
print(Cs.getCount())

File: class_member/3.rb
class Cs
  @@count = 0
  def initialize
    #@@는 클레스에 속하는 값
    @@count = @@count+1
  end
  def Cs.getCount
    return @@count
  end
end
i1 = Cs.new
i2 = Cs.new
i3 = Cs.new
i4 = Cs.new
p Cs.getCount
